scalafix-tests/input/src/main/scala-2/test/explicitResultTypes/ExplicitResultTypesBase.scala

```diff
@@ -36,6 +36,7 @@ object ExplicitResultTypesBase {
     x + 2
   object ExtraSpace {
     def * = "abc".length
+    def ! = "abc".length
     def foo_ = "abc".length
     def `x` = "abc".length
     def `x ` = "abc".length
```

Comment:
This case is not working because semanticdb symbol for this is:

`x` => test/explicitResultTypes/ExplicitResultTypesBase.ExtraSpace.x().
`x ` => test/explicitResultTypes/ExplicitResultTypesBase.ExtraSpace.x(+1).

The inverse method is too naive with this and the methhod fails because the type has following members:

`List(<init>, x, x$u0020)`

We can't easily map `x(+1)` into `x$u0020`. @tgodzik pinging you to let you know